lib/plugins/pumpiob.js

```diff
@@ -0,0 +1,71 @@
+/* jshint node: true */
+'use strict';
+
+var _ = require('lodash');
+
+function init() {
+
+  var pumpiob = {
+    name: 'pumpiob'
+    , label: 'Pump IOB'
+    , pluginType: 'pill-major'
+  };
+
+  pumpiob.setProperties = function setProperties(sbx) {
+    sbx.offerProperty('pumpiob', function setPumpIOB () {
```

Comment:
Does the `iob` plugin take basal into account? Pump IOB considers bolus only (@scottleibrand actually suggested calling this something like bolus-on-board to distinguish them), so it would result in different calculations from other plugins if so.